### PR TITLE
add question to the html

### PR DESCRIPTION
**Pull request by alpsz**

### CODE REVIEW

File: html/answers/alps-q4.html [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        gtag('config', 'UA-157837769-1');
+        </script>
+        
+        <title>
+            HTML | InterQues | Coding Ninjas
+        </title>
+
+        <link rel="shortcut icon" type="image/png" href="../../assets/favicon.png"/>
+
+        <meta name="description" content="What is semantic HTML?">
```
Comment (shivamyth) [inline html/answers/alps-q4.html:19]: You have added many unnecessary things. Correct it and resend the PR.

**Review by shivamyth — CHANGES_REQUESTED** [review]

File: html/answers/alps-q4.html [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+                </p>
+               
+            </article>
+        </content>
+
+        <hr>
+
+        <div>
+            <h5>
+                Similar Questions
```
Comment (dakshkhetan) [inline html/answers/alps-q4.html:102]: @alpsz "Similar Questions" is written two times in this file i.e. some lines of code have been duplicated/repeated.
Please make changes to your answer file ASAP.

**Review by dakshkhetan — CHANGES_REQUESTED** [review]
@alpsz Please make the requested changes as this PR will be closed in 24 hours if no changes are made.

**Comment by dakshkhetan** [comment]
PR closed due to no response.

**Pull request closed without merging**